shared/src/main/scala/com/raquo/domtypes/generic/defs/tags/SvgTags.scala

```diff
@@ -0,0 +1,733 @@
+package com.raquo.domtypes.generic.defs.tags
+
+import com.raquo.domtypes.generic.builders.TagBuilder
+
+trait SvgTags[T[_ <: DomElement], DomElement, SvgElement <: DomElement,
+    SvgCircle <: SvgElement,
```

Comment:
Oof, that's a lot of type params. I looked into a few of those and it seems that they do provide value, thanks for digging those out.

It would be great to batch-rename all these type params to have SvgElement suffix for consistency with other traits. For example: `SvgComponentTransferFunction` -> `ComponentTransferFunctionSvgElement`